cellprofiler/modules/blobdetection.py

```diff
@@ -0,0 +1,379 @@
+# coding=utf-8
+
+"""
+
+Find blobs in a grayscale (single-channel) image or volume. Blobs are bright on dark or dark on bright regions in an
```

Comment:
I thought about using `rgb2gray`, but that assumes all multi-channel images are RGB images.